Gemfile

```diff
@@ -25,7 +25,7 @@ gem 'validates_email_format_of', '1.6.3' # validates email against RFC 2822 and
 
 # 0.7.3 is the latest for Rails 4.2, however, it is absent on Rubygems server
 # https://github.com/huacnlee/rails-settings-cached/issues/165
-gem 'nokogiri', '~> 1.12.5'
+gem 'nokogiri', '~> 1.12', '>= 1.12.5'
```

Comment:
Also, problems with updates must also be solved in a separate branch, it is not known when your branch will get into the master. Besides, the problem may be different. And so, for example, I may have an old version of nokogri and my PR will go to the master first of yours, and when it’s your turn to branch, a conflict may arise

Reply:
Actually it is nothing to do with these settings, we already have the last version of nokogiri. 2 days ago ruby 3.1.0 released, but nikogiri cannot yet support this version